Cargo.toml

```diff
@@ -57,7 +58,7 @@ trust-dns-resolver = { version = "0.20.0" , optional = true, default-features =
 
 [dev-dependencies]
 doc-comment = "0.3"
-futures-channel = { version = "0.3.7", default-features = false, features = ["sink"] }
+futures-util = { version = "0.3.7", default-features = false }
```

Comment:
```suggestion
futures-util = { version = "0.3.8", default-features = false }
```

I'm aware it doesn't really matter but since 0.3.8 is the current version, we might as well use that. ;)

Reply:
Thanks for point out but we are using 0.3.7 across all the actix-* crates.
There is no lock on the crates so users can choose to use anything above 0.3.7

Reply:
Yep, no need to needlessly bound the version in a library when we don't need to. It's ^0.3.7 to avoid an security warning. Eg: https://deps.rs/crate/actix-web/3.3.2

Reply:
I'm not saying we should fix the version, rather the opposite. I actually would even prefer going 0.3 instead of specifying the minor. But by specifying 0.3.8 we're forcing all dependencies to go at least to 0.3.8, without `cargo update` which is a good thing in my book.

Reply:
Allowing versions below 0.3.7 is a security risk and shows up on our deps.rs page as such.

Reply:
There was a UB for waker fixed in 0.3.6. So it's suggested to use at least that. It's always good to do our part on this matter.

Reply:
> Allowing versions below 0.3.7 is a security risk and shows up on our deps.rs page as such.
> 
> FYI @therealprof: https://github.com/RustSec/advisory-db/blob/master/crates/futures-util/RUSTSEC-2020-0059.md

I don't understand what you're trying to tell me, sorry. I would like to go (higher, i.e. 0.3.8), not lower.

Reply:
In terms of maintaining a library (rather than an app), allowing the most versions of crates in a dep tree makes it most compatible and gives developers the option to upgrade as they decide or, more importantly, pin a lower compatible version because of a bug/yanked crate/security issue or whatever else.